api/v1beta1/cinderapi_types.go

```diff
@@ -51,6 +51,12 @@ type CinderAPITemplate struct {
 	ContainerImage string `json:"containerImage"`
 
 	CinderAPITemplateCore `json:",inline"`
+
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=4
+	// +kubebuilder:validation:Minimum=1
+	// Workers - Number of processes running CinderAPI
+	Workers *int32 `json:"workers"`
```

Comment:
@ASBishop as this is a parameter we tune for `CinderAPI`, my idea is to keep it under `CinderAPITemplate`: it results clear how to access this value from `cinder_controller`. Do you think we have a better place to expose this value?
FYI I'm doing the same thing for `Manila`, I'd like to keep the two operators in sync.